javaparser-core/src/main/java/com/github/javaparser/ast/expr/StringLiteralExpr.java

```diff
@@ -78,6 +78,10 @@ public boolean remove(Node node) {
         return super.remove(node);
     }
 
+    public String asString() {
+        return value;
+    }
```

Comment:
Are there any cases where you want the unescaped value? If there is no demand for special treatment of escaped and unescaped value, maybe we should escape the 'value' in the constructor? and return in the asXXX methods always the escaped one?